config/routes.rb

```diff
@@ -81,7 +81,14 @@
 
   # You can have the root of your site routed with "root"
   # just remember to delete public/index.html.
-  root :to => 'home#index'
+
+  # MLN application supports multiple domains.
+  match 'about/participating-schools' => 'info_site#participating_schools', via: [:get], :constraints => { :host => ENV['MLN_URL'] }
+  match 'about/about-mylibrarynyc' => 'info_site#about', via: [:get], :constraints => { :host => ENV['MLN_URL'] }
+  match '/contacts-links' => 'info_site#contacts', via: [:get], :constraints => { :host => ENV['MLN_URL'] }
+
+  root :to => "home#index", :constraints => {:host => ENV['MLN_SETS_URL']}
+  root :to => 'info_site#index', :constraints => {:host => ENV['MLN_URL']}
```

Comment:
You could maybe move line 91 to the top of the block starting on line 86. That would keep all the info site routes grouped together. Then, I _think_ you wouldn't need the `ENV['MLN_SETS_URL']` constraint on line 91. And in fact maybe you could get away from defining `ENV['MLN_SETS_URL']` at all.